tests/methodologies/para/test_ai_heuristic.py

```
@@ -1507,14 +1456,28 @@ class MockStat:
         result = h.evaluate(f)
 
         # stable_reference signal should be present
         resource_signals = result.scores[PARACategory.RESOURCE].signals
         assert "stable_reference" in resource_signals
         assert result.scores[PARACategory.RESOURCE].score > 0
+
+    def test_stat_provider_oserror_returns_neutral_result(self, tmp_path: Path) -> None:
+        """stat_provider raising OSError must return needs_manual_review, not propagate."""
+        from methodologies.para.detection.heuristics import TemporalHeuristic
+
+        def _raising_stat(path: object) -> object:
```

Ruff (0.15.10)

[warning] Unused function argument: `path`

(ARG001)